doc/DS18S20.md

```diff
@@ -0,0 +1,23 @@
+Dallas Semiconductors DS18S20 temperature sensor
+================================================
```

Comment:
I just copied the docs from the st7735fb repo ... I created a breadboard with this LCD, some switches and a temperature sensor on it. Will tidy it all up when I write up the docs